pgmpy/base/DAG.py

```diff
@@ -373,7 +373,7 @@ def active_trail_nodes(self, variables, observed=None):
 
         Examples
         --------
-        >>> from pgmpy.models import BayesianModel
+        >>> from pgmpy.base import DAG
         >>> student = BayesianModel()
```

Comment:
>>> student = BayesianModel()
Should be:
 >>> student = DAG()

Reply:
Fixed